client/stylesheets/abstracts/_colors.scss

```diff
@@ -2,3 +2,12 @@
 
 @import 'node_modules/@automattic/color-studio/dist/color-variables.scss';
 @import 'node_modules/@wordpress/base-styles/colors.scss';
+
+// This is temp placeholders for missing variables after styles upgrade
+// Values taken from: https://github.com/WordPress/gutenberg/blob/%40wordpress/base-styles%401.0.0/packages/base-styles/_colors.scss
+// TODO: Replace variables with new color palette
+$light-gray-500: #e2e4e7;
+$light-gray-200: #f3f4f5;
+$light-gray-300: #edeff0;
+$dark-gray-900: #191e23;
+$dark-gray-500: #555d66;
```

Comment:
@LevinMedia those color variables are removed from latest `@wordpress/base-styles`. Should we keep them as it is done now or replace with alternatives from `@automattic/color-studio`?

Links: [old base-styles (v1.0.0)](https://github.com/WordPress/gutenberg/blob/%40wordpress/base-styles%401.0.0/packages/base-styles/_colors.scss), [current base-styles (v3.4.1)](https://github.com/WordPress/gutenberg/blob/%40wordpress/base-styles%403.4.1/packages/base-styles/_colors.scss).

Reply:
Replaced one by one in commits from https://github.com/Automattic/woocommerce-payments/pull/1536/commits/2f531cee48e97f2ddc840e75a89c969be5892086 to https://github.com/Automattic/woocommerce-payments/pull/1536/commits/b1fe1051def5f6075cee5a9ec2b1e522a795b888